docs/extensions/cybersecurity_profile.md

```diff
@@ -0,0 +1,39 @@
+# Cybersecurity Profile
+
+This profile can be used to specify how VSS signals can be extended to include cybersecurity attibutes.
+
+## Mapping Syntax
+
+The syntax for definition of a signal with cybersecurty attributes (in an overlay) is specified below.
+
+```yaml
+<VSS Signal name>:
+  type: <VSS type>
+  datatype: <VSS datatype>
+  cybersecurity:
+    [authentication: {true|false}]
+    [encryption: {true|false}]
+    [integrity_check: {true|false}]
+    [impact_profiles: {[{motion|privacy|regulation|safety|operational}]}]
+```
+- Set `authentication` attribute to true if you need to ensure serialized data is being exchanged between trusted entities or prevent unauthorized access to serialized data during transmission. Sample implementation mechanisms are public-key infrastructure, token based authentication etc.
```

Comment:
Is the intention that this attribute covers both scenarios where only sender is authenticated and scenarios with mutual authentication?